landscape.yml

```diff
@@ -852,25 +852,25 @@ landscape:
         items:
           - item:
             name: Sony Imageworks
+            second_path: Dev Days 2024 / Supporting Company
```

Comment:
Could you use an array of strings instead of a single string when using `second_path` @jmertic? This is breaking the ASWF landscape v2 build 😇

https://github.com/cncf/landscape2-sites/actions/runs/9856659544/job/27214136045

This applies to some more entries below as well. Thanks!